README.md

```diff
@@ -163,7 +170,6 @@ prebuild [options]
   --compile     -c              (compile your project using node-gyp)
   --no-compile                  (skip compile fallback when downloading)
   --libc                        (use provided libc rather than system default)
-  --backend                     (specify build backend, default is 'node-gyp')
```

Comment:
I guess if we want to keep ninja we need to include it as a dependency again since it isn't bundled with npm.
But ninja requires node-gyp so we can't remove it and won't gain any improvement in package size.